### ISSUE TITLE
Version for Windows qubes

### ISSUE THREAD

**Issue by Atrate** [iblue/qvm-mount#1]
It'd be nice if there was a version of this utility that worked for Windows qubes as clients as well.

**Comment by iblue** [comment]
This is true. Feel free to write one, I'll be happy to review your pull request.
